core/tests/test_utils.py

```diff
@@ -73,11 +73,26 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """ Sets the content, status code and headers that the urlfetch mock
+        should return with.
+
+        Args:
+            content: str. The content to return in subsequent calls to the
+            urlfetch mock.
+            status_code: int. status_code for return_values.
+            headers: headers to return_values.
+        """
         self.return_values['content'] = content
         self.return_values['status_code'] = status_code
         self.return_values['headers'] = headers
 
     def _Dynamic_Fetch(self, request, response): # pylint: disable=invalid-name
+        """ Simulates a url fetch.
+
+        Args:
+            request: Request object for the mock url.
```

Comment:
Missing type info. Please check that this is specified for each arg and return value.